tests/Functional/Security/Http/Logout/ContextInvalidationLogoutHandlerTest.php

```diff
@@ -16,13 +16,19 @@
 use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 use Symfony\Component\BrowserKit\Cookie;
 use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Component\Security\Http\Event\LogoutEvent;
 
 class ContextInvalidationLogoutHandlerTest extends WebTestCase
 {
     use MockeryPHPUnitIntegration;
 
     public function testLogout()
     {
+        if (class_exists(LogoutEvent::class)) {
+            // @see https://github.com/symfony/symfony/pull/36243/files#r465083756
+            $this->markTestSkipped('This test does not work with Symfony 5.1.');
```

Comment:
lets wait until tomorrow if we get a reaction on for https://github.com/symfony/symfony/pull/36243/files#r465083756 and then either remove this again, or also add a check to the actual implementation to make people realize the handler does not work with symfony 5.1

Reply:
Are you sure you want to throw an exception when the class `ContextInvalidationLogoutHandlerTest` is used in Symfony 5.1? I'm little bit afraid doing that 🙈

Reply:
i think its the correct thing to do. its not something we can silently work around, it simply does not work at all so devs need to be made aware if they did not check the changelog. better a clear error than a security issue that is silently sitting there.

Reply:
Done in https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/545/commits/853eaa0ea8bdcbf44769fc935f680dd843d189e0